README.md

```diff
@@ -397,99 +517,193 @@ Also if you like to automatically record the execution time of different cells
+```
 
-see section *Using Docker compose* to fire up the Hapi fhir server and openmrs.  
-Note that Openhim listens to client requests at `port:5001` by default. 
+see section *Using Docker compose* to fire up the Hapi fhir server and openmrs.
+\
+Note that Openhim listens to client requests at `port:5001` by default.
 
-you can now start the pipeline with these args below to point to the OpenHIm instance
+you can now start the pipeline with these args below to point to the OpenHIm
+instance
 
-    `--sinkUserName=hapi --sinkPassword=Admin123 --fhirSinkPath=http://localhost:5001/fhir` 
+```
+`--sinkUserName=hapi --sinkPassword=Admin123 --fhirSinkPath=http://localhost:5001/fhir`
+```
 
-ie 
+ie
 
 for batch
 
-    mvn compile exec:java -pl batch \
-        "-Dexec.args=--openmrsServerUrl=http://localhost:8099/openmrs  --searchList=Patient \
-        --batchSize=20 --targetParallelism=20  \
-        --fhirSinkPath=http://localhost:5001/fhir \
-        --sinkUserName=hapi --sinkPassword=Admin123 \
-        --openmrsUserName=admin --openmrsPassword=Admin123 "
+```
+mvn compile exec:java -pl batch \
+    "-Dexec.args=--openmrsServerUrl=http://localhost:8099/openmrs  --searchList=Patient \
+    --batchSize=20 --targetParallelism=20  \
+    --fhirSinkPath=http://localhost:5001/fhir \
+    --sinkUserName=hapi --sinkPassword=Admin123 \
+    --openmrsUserName=admin --openmrsPassword=Admin123 "
+```
+
+for streaming
 
-for streaming 
+```
+mvn compile exec:java -pl streaming-binlog \
+  -Dexec.args="--databaseHostName=localhost \
+  --databasePort=3306 --databaseUser=root --databasePassword=debezium\
+  --databaseName=mysql --databaseSchema=openmrs --databaseServerId=77 \
+  --openmrsUserName=admin --openmrsPassword=Admin123 \
+  --openmrsServerUrl=http://localhost:8099/openmrs \
+  --fhirSinkPath=http://localhost:5001/fhir \
+  --sinkUserName=hapi --sinkPassword=Admin123 \
+  --fileParquetPath=/tmp/"
+```
 
-    mvn compile exec:java -pl streaming-binlog \
-      -Dexec.args="--databaseHostName=localhost \
-      --databaseUser=root --databasePassword=debezium\
-      --databaseSchema=openmrs \
-      --openmrsUserName=admin --openmrsPassword=Admin123 \
-      --openmrsServerUrl=http://localhost:8099/openmrs \
-      --fhirSinkPath=http://localhost:5001/fhir \
-      --sinkUserName=hapi --sinkPassword=Admin123 \" 
-      
+You can track all the transactions in the OpenHIM instance Under the Tab
+`Transaction Log`
 
-You can track all the transactions in the OpenHIM instance Under the Tab `Transaction Log`
+see demo at https://youtu.be/U1Sz3GUKbIw
 
-see demo at  https://youtu.be/U1Sz3GUKbIw
+see
+https://openhim.readthedocs.io/en/latest/how-to/how-to-setup-and-configure-openhim.html
+for more details about setting up OpenHIM
 
-see https://openhim.readthedocs.io/en/latest/how-to/how-to-setup-and-configure-openhim.html for more details about setting up OpenHIM
+## Running end-to-end tests
```

Comment:
And let's the next two sections to two other separate .md files under `docs/`.

Reply:
Will do in follow-up PR.